packages/lib/shim-init-node.js

```diff
@@ -16,6 +16,7 @@ const https = require('https');
 const toRelative = require('relative');
 const timers = require('timers');
 const zlib = require('zlib');
+const Response = require('node-fetch').Response;
```

Comment:
I'm new to node and modern JS. Is this acceptable or is it preferred to do `const { Response } = require('node-fetch');` which I think should have the same result?

Reply:
I don't mind either way but I'd do the second option as it avoids a bit of redundant code.